pkg/reconciler/build/build.go

```diff
@@ -154,7 +188,7 @@ func (c *Reconciler) reconcile(ctx context.Context, build *buildapi.Build) error
 			})
 
 			if err != nil {
-				return errors.Wrap(err, "unable to create app image keychain")
+				return fmt.Errorf("unable to create app image keychain: %v", err)
```

Comment:
Why do  we choose to stringify the message instead of wrapping it?  Do we wish to drop the stack trace here?